toolkit/scripts/tools.mk

```diff
@@ -148,11 +149,11 @@ worker_chroot_deps := \
 	$(PKGGEN_DIR)/worker/create_worker_chroot.sh
 
 ifeq ($(REFRESH_WORKER_CHROOT),y)
-$(chroot_worker): $(worker_chroot_deps) $(depend_REBUILD_TOOLCHAIN) $(depend_TOOLCHAIN_ARCHIVE)
+$(chroot_worker): $(worker_chroot_deps) $(go-bldtracker) $(depend_REBUILD_TOOLCHAIN) $(depend_TOOLCHAIN_ARCHIVE)
 else
-$(chroot_worker):
+$(chroot_worker): $(go-bldtracker)
```

Comment:
Why is `go-bldtracker` a dependency for the chroot worker? And if it is, then it should go into `worker_chroot_deps`.

Reply:
Please still move `go-bldtracker` into `worker_chroot_deps` above so we're consistent.

Reply:
I think we do want this tool broken out for both, since there is a case where we are asking Make to skip the chroot tools, but the file is missing. In this case it will attempt to rebuild... thinking on this maybe this is just a shortcoming of `REFRESH_WORKER_CHROOT` in general since it will be missing a bunch of files like the toolchain rpms.

Reply:
`REFRESH_WORKER_CHROOT` is a workaround we'd like to remove at some point and using it assumes you have already have the chroot built, so the tool should be available as well. I don't want to add any dependencies for this `else` case on purpose, to not encourage people to get used to the switch and assume it's there to stay.

Back to `$(go-bldtracker)` - could we still move it inside `worker_chroot_deps` above? That's where we keep these dependencies after all.